fxa-oauth-server/lib/routes/token.js

```diff
@@ -162,7 +162,7 @@ module.exports = {
       access_token: validators.token.required(),
       refresh_token: validators.token,
       id_token: validators.assertion,
-      scope: validators.scope.required(),
+      scope: validators.scope.required().allow(''), // YA!
```

Comment:
I'm going to try changing the scope regex to use `*` rather than `+`, so we accept the empty string everywhere we validate a scope.

Reply:
this change should go away, fix in the other PR to the regex.